docs/src/demos/components/Columns/stretch-columns.demo.tsx

```diff
@@ -1,7 +1,7 @@
 import type { ReactNode } from 'react';
 import { Box, Columns, Stack } from '@marigold/components';
 
-const Block = ({
+export const Block = ({
```

Comment:
should this look like this?

<img width="815" alt="Bildschirmfoto 2022-09-21 um 08 27 29" src="https://user-images.githubusercontent.com/38324334/191430233-a2df7b93-008d-4a31-8c02-a200325dec48.png">